tests/test_fmi_imports.py

```diff
@@ -0,0 +1,110 @@
+import pytest
+import pyfmi.fmi as fmi
+
+@pytest.mark.parametrize("x",
```

Comment:
Will remove, but here for visibility; these are all the imports one could get from `pyfmi.fmi`, i.e., nothing broke.